### ISSUE TITLE
Missing Add-ons

### ISSUE THREAD

**Issue by ojasva** [HaikuArchives/Becasso#27]
After reading the Manual, I see that the author developed add-ons for various features. However, they aren't available right now. I came across a paragraph where author requested 29$ for the full functionality. I suspect that could be the reason for missing functionalities.

**Comment by janus2** [comment]
The add-ons are in source/add-ons probably they are not packaged correctly.

**Comment by thaflo** [comment]
Yes, they work. After making the project, they are in subfolders in "add-ons".
The compiled add-ons have to be copied in /add-ons, then they are loaded

**Comment by thaflo** [comment]
Yes, they work. After making the project, they are in subfolders in "add-ons".
The compiled add-ons have to be copied in /add-ons, then they are loaded

after you do "jam install" -> you'll get a folder named "target" with Becasso and it's addons.
